WalletWasabi.Fluent/Controls/Dialog.axaml

```diff
@@ -56,12 +56,34 @@
     </Setter>
     <Setter Property="Opacity" Value="0" />
   </Style>
+
   <Style Selector="c|Dialog /template/ Border#PART_Border">
     <Setter Property="Margin" Value="40" />
     <Setter Property="BorderThickness" Value="1" />
     <Setter Property="CornerRadius" Value="4" />
     <Setter Property="BoxShadow" Value="{DynamicResource DialogBoxShadow}" />
   </Style>
+
+  <Style Selector="c|Dialog:alert /template/ Border#PART_Border">
+    <Setter Property="BoxShadow" Value="{DynamicResource DialogBoxShadow}"/>
+    <Style.Animations>
+      <Animation Duration="0:0:0.8" PlaybackDirection="AlternateReverse" IterationCount="1">
+        <KeyFrame Cue="0%">
+          <Setter Property="BoxShadow" Value="{DynamicResource DialogAlertStartBoxShadow}"/>
+        </KeyFrame>
+        <KeyFrame Cue="25%">
+          <Setter Property="BoxShadow" Value="{DynamicResource DialogAlertEndBoxShadow}"/>
+        </KeyFrame>
+        <KeyFrame Cue="75%">
+          <Setter Property="BoxShadow" Value="{DynamicResource DialogAlertStartBoxShadow}"/>
+        </KeyFrame>
+        <KeyFrame Cue="100%">
+          <Setter Property="BoxShadow" Value="{DynamicResource DialogBoxShadow}"/>
+        </KeyFrame>
+      </Animation>
+    </Style.Animations>
+  </Style>
```

Comment:
I would adjust the animation a tiny bit, it feels too slow. It might be only a personal preference so only apply this if someone else agrees too.

Current one:
![dialog_default](https://user-images.githubusercontent.com/16364053/222668094-d45ff11d-f962-48e2-b385-19533ce5d1fa.gif)

Adjusted:
![dialog_adjusted](https://user-images.githubusercontent.com/16364053/222668128-1ae5e5b6-07d7-4019-9427-201f96113b12.gif)

```xml
<Animation Duration="0:0:0.6" IterationCount="1">
  <KeyFrame Cue="25%">
    <Setter Property="BoxShadow" Value="{DynamicResource DialogAlertStartBoxShadow}"/>
  </KeyFrame>
  <KeyFrame Cue="50%">
    <Setter Property="BoxShadow" Value="{DynamicResource DialogAlertEndBoxShadow}" />
  </KeyFrame>
  <KeyFrame Cue="75%">
    <Setter Property="BoxShadow" Value="{DynamicResource DialogAlertStartBoxShadow}"/>
  </KeyFrame>
  <KeyFrame Cue="100%">
    <Setter Property="BoxShadow" Value="{DynamicResource DialogBoxShadow}"/>
  </KeyFrame>
</Animation>
```

Start BoxShadow: 0 0 10 5
End BoxShadow: 0 0 10 1

Reply:
Adjusted animation and box shadow